cli/Makefile

```
@@ -1,17 +1,31 @@
+	XCADDY_FLAGS := -gcflags='all=-N -l' -tags=nobadger,nomysql,nopgx,nodphp,nobrotli
+else
+	XCADDY_FLAGS := -ldflags='-w -s' -tags=nobadger,nomysql,nopgx,nodphp,nobrotli
+endif
+
+LOCAL_MODULE := /home/withinboredom/code/durable-php/cli
+GEN_STUB := /home/withinboredom/code/php-src/build/gen_stub.php
+LOCAL_FRANKENPHP_CADDY := /home/withinboredom/code/frankenphp/caddy
+LOCAL_FRANKENPHP := /home/withinboredom/code/frankenphp
+
+# Targets
+frankenphp: ext/build/ext.go ext/build/ext_arginfo.h
```

checkmake (0.2.2)

[warning] Target body for "frankenphp" exceeds allowed length of 5 (9).

(maxbodylength)